Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -336,7 +337,8 @@ class AccountStatusSetting: WithAccountSetting {
 
     override func onClick(_ navigationController: UINavigationController?) {
         guard !profile.rustFxA.accountNeedsReauth() else {
-            let view = FxAWebView(pageType: .emailLoginFlow, profile: profile, dismissalStyle: .popToRootVC)
+            let view = FxAWebViewController(pageType: .emailLoginFlow, profile: profile, dismissalStyle: .popToRootVC)
```

Comment:
I expected `FirefoxAccountSignInViewController` here, please double check this

Reply:
Will change this!